src/adapters/github.js

```diff
@@ -146,10 +146,13 @@ class GitHub extends PjaxAdapter {
       return cb();
     }
 
+    // Trivial check whether typeId is a commit id in case URL '.../tree/{commitId}'
+    const isCommit = type === 'commit' || !!typeId.match(/[a-z, 0-9]{40}/)
```

Comment:
Not sure if we need the 2nd check. I think it could just be

```
const branch = 
   // Special handling for commit
   type === 'commit' && typeId ||
   // Code page
   ...